doc/installation_instructions/basic.rst

```diff
@@ -0,0 +1,300 @@
+******************
+Basic Installation
+******************
+
+The following will show how to install both SmartSim and SmartRedis.
+
+.. note::
+  For users on platforms with a 'site install' of SmartSim please follow
+  :ref:`these instructions <site install>`.
+
+=============
+Prerequisites
+=============
+
+The base prerequisites to install SmartSim and SmartRedis are:
+
+  - Python 3.7-3.9
+  - Pip
+  - Cmake 3.13.x (or later)
+  - C compiler
+  - C++ compiler
+  - GNU Make > 4.0
+  - git
+  - git-lfs
+
+.. note::
+
+  GCC 5-9, 11, and 12 is recommended. There are known bugs with GCC 10.
+
+The machine-learning backends have additional requirements
+
+  - CUDA Toolkit 11 (11.4 recommended) `Instructions <https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html>`_
```

Comment:
Should you specify that some of these tool kits are not required when running via CPU? That might just be for the scaling-study repo tho